.vscode/launch.json

```diff
@@ -8,11 +8,36 @@
             "name": ".NET Core Launch (web)",
             "type": "coreclr",
             "request": "launch",
-            "preLaunchTask": "build",
+            "preLaunchTask": "buildweb",
             // If you have changed target frameworks, make sure to update the program path.
-            "program": "${workspaceFolder}/src/Relecloud.Web/bin/Debug/net6.0/Relecloud.Web.dll",
+            "program": "${workspaceFolder}/src/Relecloud.Web.CallCenter/bin/Debug/net6.0/Relecloud.Web.CallCenter.dll",
```

Comment:
@twsouthwick I believe this should be updated to be net8.0 now, could you please confirm? @PabloNunes we recently had to bump our minimum .NET version due to package support requirements.